### ISSUE TITLE
stun server stun.stunprotocol.org seems to be down

### ISSUE THREAD

**Issue by showstoppre** [jselbie/stunserver#50]
Binding fails when I try a stun request to [stun.stunprotocol.org](http://stun.stunprotocol.org/)

**Comment by jselbie** [comment]
In recent months, stun.stunprotocol.org was exceeding hosting cost limits - by a lot.  Keeping the lights on globally for this service would cost me about $4000/year in bandwidth costs alone. In previous years, it was only a few hundred dollars.

It's a free service meant for showcasing the open source code and for individuals to use for personal use. But over the years, it's been plagued with denial of service attacks. And most recently, some organizations appeared to have deployed a large scale production service hardcoded to use it. 

So last week, I made the harsh decision to limit it to only North America and Europe

The TLDR is that if you need a personal stun server, use the new address: **stunserver.stunprotocol.org**.

**Comment by showstoppre** [comment]
Thanks for letting me know. I'll use the new address. I use it only for personal purposes, ~10 reqs per day.

**Comment by jselbie** [comment]
@showstoppre - 10 is fine.   10 million gets to be a problem - as some folks are attempting to do. :)
